.drone/scripts/build-native.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -e
+
+git fetch
+export VERSION=$(cat .data.json | jq -r '.current_pkgver')
+tar -cJf ../makedeb_$VERSION.orig.tar.xz .
+debuild -us -uc
```

Comment:
Are we gonna be automatically publishing the Debian package too, or are we gonna be doing that in a future commit?